debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunner.kt

```diff
@@ -60,7 +60,7 @@ class RsDebugRunner : com.intellij.execution.runners.AsyncGenericProgramRunner<R
     override fun prepare(env: ExecutionEnvironment, state: RunProfileState): Promise<RunProfileStarter> {
         val cleaned = (env.runnerAndConfigurationSettings!!.configuration as CargoCommandConfiguration).clean().ok!!
         val cargoProjectDirectory = cleaned.cargoProject.manifest.parent
-        val cargo = env.project.toolchain!!.cargo(cargoProjectDirectory)
+        val cargo = env.project.toolchain!!.cargoOrXargo(cargoProjectDirectory)
```

Comment:
if I recall correctly recently something called `wargo` was released

https://github.com/lord/wargo

though it doesn't look like another cargo replacement, nevertheless the name `cargoOrXargo` might become obsolete soon